apps/src/pcd_select_object_plane.cpp

```diff
@@ -638,9 +638,7 @@ main (int argc, char** argv)
     return (-1);
   }
   
-  string object_file = "object.pcd", plane_file = "plane.pcd", rest_file = "rest.pcd";
-  if (p_file_indices.size () >= 4)
-    rest_file = argv[p_file_indices[3]];
+  string object_file = "object.pcd", plane_file = "plane.pcd";
```

Comment:
Please [isolate the declaration](https://clang.llvm.org/extra/clang-tidy/checks/readability-isolate-declaration.html) to increase the readability.